scenarios/scenario-17/README.md

```diff
@@ -75,6 +75,33 @@
    ```
 <br>
 
+#### How to expose an API using a different Ingress Hostname
+
+- When you are creating the API you can define a hostname if required. If a hostname is not defined
+it takes the default ingress hostname (mgw.ingress.wso2.com).
+
+   ```
+        >> apictl add api -n hello-world-api --from-file=swagger.yaml --override --hostname=mgw.group1.wso2.com
```

Comment:
Shall we unindent this command

Reply:
Fixed with 5bda6eea84765c8ec5a0dc189fbc3c5c437aacb5